lib/test_suite.py

```diff
@@ -45,6 +45,9 @@ class TestSuite:
     server for this suite, the client program to execute individual
     tests and other suite properties. The server is started once per
     suite."""
+
+    RETRIES_COUNT = 3
```

Comment:
It means 4 runs (3 retries). Is it expected or you supposed to set the overall amount of runs to 3?

Reply:
It is expected. 4 runs (3 retries).